manim/config.py

```diff
@@ -239,3 +240,47 @@ def get_camera_configuration(args):
         camera_config["background_opacity"] = 0
 
     return camera_config
+
+
+def initialize_directories(cfgargs):
```

Comment:
Please use a more readable name; I only used `cfg` in `__init__.py` because `config` was taken by the import.

Reply:
Can do, but what'd be a good variable name here? config_args? Doesn't seem much better to me either.

Reply:
Sure. Done.